lib/index.js

```diff
@@ -35,6 +35,24 @@ const appendFood = (food) => {
       </article>`)
 }
 
+const addNewFood = () => {
+  event.preventDefault()
+  var data = { name: $('#new-name').val(), calories: $('#new-calories').val()}
```

Comment:
I would pull this out to a separate method

Reply:
I think this is probably the only place we'll use this so I think we should keep it in the function. It's also just defining a variable, just looks kind of long! Let me know what you think.

Reply:
ok great! good point